packages/vaadin-lumo-styles/mixins/required-field.js

```diff
@@ -10,17 +10,26 @@ import '../typography.js';
 import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const requiredField = css`
+  :host {
+    /* Label properties */
+    --_label-color-default: var(--vaadin-input-field-label-color, var(--lumo-secondary-text-color));
+    --_label-color-focused: var(--vaadin-input-field-focused-label-color, var(--lumo-primary-text-color));
+    --_label-color-hovered: var(--vaadin-input-field-hovered-label-color, var(--lumo-body-text-color));
+    --_label-font-size: var(--vaadin-input-field-label-font-size, var(--lumo-font-size-s));
+    --_label-font-weight: var(--vaadin-input-field-label-font-weight, 500);
```

Comment:
Good question. We don't have these on the preliminary list at the moment, although some other `--lumo-` prefixed properties especially `--lumo-button-size` and `--lumo-text-field-size` are planned to be aligned somehow.

Reply:
Indeed, I had forgotten about `--lumo-required-field-indicator` and `--lumo-required-field-indicator-color`.

I think that now that we're drawing a line between the abstract Lumo properties prefixed with `--lumo-` and the component/feature-specific properties prefixed with `--vaadin-`, it would make sense to refactor those to be `--vaadin`-prefixed (while keeping the current props around for backward compact, of course)